src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import './styles/index.css'
 import '@fontsource/dm-sans/index.css'
 import 'react-tabs/style/react-tabs.css'
-import './bootstrap'
+import './bootstrap.ts'
```

Comment:
why does it needed ? doesn't builder runs through all possible variants if module extension omitted (e.g. js -> jsx -> ts -> tsx -> mjs etc)

Reply:
there was a name conflict. i'll retest if still required